### DISCUSSION TITLE
processgui README update django?

### DISCUSSION THREAD

**Discussion by rmchurch** [ukaea/PROCESS#3030] [Q&A] [answered]
Trying to run `start_gui`. After installing the newest version of django, and finding they don't care at all about breaking changes, I saw the README saying 1.7. However that has an `HTMLParseError` not being found. I upgrade to 1.8 seems to work so far. Is `start_gui` still the preferred way for setting up input files?

**Comment by mkovari** [comment]
Hi Michael.  Welcome to PROCESS.  It seems you must be looking at the user guide from ~2016?  I am afraid that no gui is currently available for setting up input files.

If you let us know what sort of tokamak or stellarator configuration you are interested in we should be able to give you some pointers.
Michael

**Comment by rmchurch** [comment]
Thanks. I was using the User Guide from the official page, https://ccfe.ukaea.uk/resources/process/, is there a newer one? It discusses the `start_gui` that is in `PROCESS/utilities/processgui/processgui/start_gui`. If the preferred way now is programmatic like as is done in the `examples/` folder, I can stick with that.

I am interested right now in stellarator configurations, even an example would be useful for getting started.

**Comment by mkovari** [comment]
Hi Michael, I'm really sorry - that page is out of date.  The latest documentation is on GitHub: https://ukaea.github.io/PROCESS/.  I will get someone in the group to give advice on stellarator input files.

**Comment by mkovari** [comment]
We currently don't have a stellarator input file that converges using the open-source version of PROCESS.  We have an older version that converges more often, and we probably could supply an input file for that.  However, that would involve getting you a licence, which would be free but involve some bureacracy.

What you might like to do is contact other people who have used PROCESS for stellarator work:
[Felix Warmer](https://www.tue.nl/en/research/researchers/felix-warmer),  f.warmer@tue.nl 
[Jorrit Lion](https://www.linkedin.com/in/jorrit-lion-546791215/), jlion@proximafusion.com
[Charles Swanson](https://www.linkedin.com/in/charles-swanson-071331153/),  [i.am.cswan@gmail.com](mailto:i.am.cswan@gmail.com)

**Comment by mkovari** [comment, marked as answer]
Overall, we don't really support stellarators at present.  Sorry about that.
Michael
